### PR TITLE
Add Vet Clinic Database Diagram

### PR DESCRIPTION
**Pull request by topeogunleye**
### Added Changes

- Added Vet Clinic Database Schema Diagram
- Added Schema based on CLinic Diagram
- Added Clinic Diagram to the README file

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  id INT GENERATED ALWAYS AS IDENTITY,
+  UNIQUE(id),
+  total_amount DECIMAL NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  payed_at TIMESTAMP NOT NULL,
+  fk_medical_history_id BIGINT NOT NULL,
+  CONSTRAINT fk_medical_history_id FOREIGN KEY (fk_medical_history_id) REFERENCES medical_histories (patient_id),
+  UNIQUE(fk_medical_history_id)
+);
+
```
Comment (SajjadAhmad14) [inline schema_based_on_diagram.sql:43]: Please create a join table for tables that have `many-to-many relationships`. So there should be a join table between `medical_histories` and `treatments`.

Please add `medical_history_id` and `treatment_id` as foreign keys in this newly created join table.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  unit_price DECIMAL NOT NULL,
+  quantity INT NOT NULL,
+  total_price DECIMAL NOT NULL,
+  invoice_id INT,
+  treatment_id INT,
+  CONSTRAINT fk_invoice_id FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+  CONSTRAINT fk_treatment_id FOREIGN KEY(treatment_id) REFERENCES treatments(id),
+  UNIQUE(invoice_id, treatment_id)
+);
+
```
Comment (SajjadAhmad14) [inline schema_based_on_diagram.sql:55]: Please add indexes to all foreign keys. This is important from the performance point of view so adding indexes to foreign keys will speed up the performance of queries.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Meltrust — APPROVED** [review]
Hi @topeogunleye,


## Wow, you did it 🎉
Brilliant
## Thank you for adding indexes to foreign keys, and for creating the join many to many table, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**